functions/openweathermap.js

```diff
@@ -5,7 +5,7 @@ const { parseJson } = require("./src/utils/misc");
 
 const fetchWeather = async () => {
   const url =
-    "https://api.openweathermap.org/data/2.5/weather?lat=40.787&lon=-119.206&units=metric&lang=en&" +
```

Comment:
do not remove, may be useful in the future,
put `const USE_IMPERIAL = true;`  just under the imports
with a @debt comment to extract it into util later on
use that flag to switch these two.

Reply:
https://github.com/sparkletown/sparkle/pull/2173#issuecomment-908932019